src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+        String split = ",";
+
+        try (BufferedReader fileReader = new BufferedReader(new FileReader(fromFileName))) {
```

Comment:
Consider using the try-with-resources statement in a more granular way. Avoid nesting the try blocks for reading and writing within the same method. It's better to separate them into different methods to handle closing resources more effectively and make the code cleaner.